modvendor/modvendor.go

```diff
@@ -109,10 +110,48 @@ func vendor(rootdir string, vendorDir string, modsrc tf.Source, report Report, i
 
 	logger.Trace().Msg("successfully downloaded")
 
-	report.addVendored(modsrc.Raw, modsrc, Dir(vendorDir, modsrc))
+	report.addVendored(modsrc, Dir(vendorDir, modsrc))
```

Comment:
don't need to change this now, but it looks like the argument could be just `modsrc` now and then we can initialize the report with the base vendordir:

```
   report := NewReport(vendordir)
   ...
   report.addVendored(modsrc)
```

Reply:
Done :100: